server/channels/db/migrations/mysql/000122_remove_upload_file_permission.up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE PROCEDURE RemoveUuploadFilePermission()
+BEGIN
+    updateRoles: LOOP
+        -- update affected rows
+        UPDATE Roles 
+            SET Permissions = REGEXP_REPLACE(Permissions, 'upload_file[[:space:]|?]', '')
+            WHERE Permissions like '%upload_file%' and Permissions not REGEXP 'create_post[[:space:]|?]'
```

Comment:
There is a permission `create_post_public`. Need to use regex, to ensure this one is not selected.